.github/workflows/build.yml

```diff
@@ -46,12 +46,31 @@ jobs:
           repository: quarkusio/quarkus
           path: ./quarkus
 
-      - name: Quarkus integration
+      - name: Quarkus integration (JAR)
         working-directory: ./quarkus
         run: |
           mvn -Dquickly
           java -jar ../target/quarkus-app/quarkus-run.jar &
           mvn clean verify -pl tcks/microprofile-lra -Dtcks -Dlra.coordinator.url=http://localhost:8080/lra-coordinator
           kill %1
 
+      - name: Quarkus integration (image)
+        run: |
+          # build the image quay.io/jbosstm/lra-coordinator:1.0.0-SNAPSHOT
+          ./mvnw clean package -Dquarkus.container-image.build=true
+          # run the built image (save logs)
+          docker run -d --rm --name lra-coordinator --network host quay.io/jbosstm/lra-coordinator:1.0.0-SNAPSHOT
+          # run the TCK in Quarkus with the produced image
+          cd ./quarkus
+          # project is already built from previous step
+          mvn clean verify -pl tcks/microprofile-lra -Dtcks -Dlra.coordinator.url=http://localhost:8080/lra-coordinator
+          # save the logs and kill the container
+          docker wait lra-coordinator
+          docker logs >"./lra-coordinator.log" 2>&1
```

Comment:
Where is this file (lra-coordinator.log) stored eventually?

Reply:
in quarkus/lra-coordinator.log. We just dump it to the console so no need to save the file.